frontend/src/components/MatchingPairs/MatchingPairs.tsx

```diff
@@ -62,6 +62,8 @@ const MatchingPairs = ({
         yPosition.current = posY;
     }
 
+    let startOfTurn = performance.now();
```

Comment:
Does it work like this in React? 🤔 If written like this, I expect `startOfTurn` to be overwritten in every re-render... I would expect a state variable that gets set on initialization and to be set again using `setStartOfTurn` in the `finishTurn` function (same place as you now do the reassignment).

Reply:
It did work but using a state variable would indeed be better.